### PR TITLE
ES-822436: Find_and_Replace_using_Regex_pattern

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Completed Find and Replace using Regex pattern documentation in Presentation

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 1]
```diff
@@ -299,6 +299,79 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-first-occurance).
 
+## Find and replace using Regex pattern 
+
+You can find either the first occurrence or all the occurrences of a pattern of text in a PowerPoint presentation using the `Find` or `FindAll` method and replace it with other text.
+
+The following code example illustrates how to find all the occurrences of a pattern of text using Regex pattern and replace it with other text.
```
Comment (florenceliliana) [inline File-Formats/Presentation/Working-with-find-and-replace.md:306]: The following code example illustrates how to find all the occurrences of a pattern of text using the Regex pattern and replace it with other text.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 2]
```diff
@@ -299,6 +299,79 @@ End Using
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Load or open an PowerPoint Presentation.
```
Comment (florenceliliana) [inline File-Formats/Presentation/Working-with-find-and-replace.md:312]: //Load or open a PowerPoint Presentation.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 3]
```diff
@@ -299,6 +299,79 @@ End Using
...
+using (FileStream inputStream = new FileStream("Sample.pptx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    //Open an existing PowerPoint presentation.
+    using (IPresentation pptxDoc = Presentation.Open(inputStream))
+    {
+        //Find all the occurrences of given pattern of text in the PowerPoint presentation using Regex.
```
Comment (florenceliliana) [inline File-Formats/Presentation/Working-with-find-and-replace.md:318]: //Find all the occurrences of a given pattern of text in the PowerPoint presentation using Regex.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 4]
```diff
@@ -299,6 +299,79 @@ End Using
...
+    }
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Opens an existing presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+    //Find all the occurrences of given pattern of text in the PowerPoint presentation using Regex.
```
Comment (florenceliliana) [inline File-Formats/Presentation/Working-with-find-and-replace.md:341]: //Find all the occurrences of a given pattern of text in the PowerPoint presentation using Regex.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 5]
```diff
@@ -447,6 +520,85 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-and-highlight-in-PowerPoint).
 
+## Find and highlight using Regex pattern 
+
+You can find either the first occurrence or all the occurrences of a pattern of text in a PowerPoint presentation using the `Find` or `FindAll` method and highlight it.
+
+The following code example illustrates how to find all the occurrences of a pattern of text using Regex pattern and highlight it.
```
Comment (florenceliliana) [inline File-Formats/Presentation/Working-with-find-and-replace.md:527]: The following code example illustrates how to find all the occurrences of a pattern of text using a Regex pattern and highlight it.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 6]
```diff
@@ -447,6 +520,85 @@ End Using
...
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Load or open an PowerPoint Presentation.
+using (FileStream inputStream = new FileStream("Sample.pptx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    //Open an existing PowerPoint presentation.
+    using (IPresentation pptxDoc = Presentation.Open(inputStream))
+    {
+        //Find all the occurrences of given pattern of text in the PowerPoint presentation using Regex.
```
Comment (florenceliliana) [inline File-Formats/Presentation/Working-with-find-and-replace.md:539]: //Find all the occurrences of a given pattern of text in the PowerPoint presentation using Regex.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 7]
```diff
@@ -447,6 +520,85 @@ End Using
...
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Opens an existing presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+    //Find all the occurrences of given pattern of text in the PowerPoint presentation using Regex.
```
Comment (florenceliliana) [inline File-Formats/Presentation/Working-with-find-and-replace.md:565]: //Find all the occurrences of a given pattern of text in the PowerPoint presentation using Regex.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The content review is completed.

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Pull request merged**